README.md

```diff
@@ -465,14 +465,12 @@ If you want to create an executable binary from this project, follow
 these steps (you'll need `go` installed and `$GOPATH` set):
 
 ``` shell
-go get gopkg.in/qntfy/kazaam.v3
-cd $GOPATH/src/gopkg.in/qntfy/kazaam.v3/kazaam
-go install
+go install https://github.com/qntfy/kazaam/kazaam@latest
```

Comment:
```suggestion
go install github.com/qntfy/kazaam/kazaam@latest
```
However, the layout of this repository means v3.4.9 is installed instead of v4.0.1 (The current latest)